nishiyama/chap1/knock02.py

```diff
@@ -0,0 +1,5 @@
+word1 = 'パトカー'
+word2 = 'タクシー'
+for i in range(0, 10):
+    print(word1[i:i+1] + word2[i:i+1], end = '')

```

Comment:
```suggestion
    print(word1[i:i+1] + word2[i:i+1], end='')
```

python（PEP8）では，空白を入れるべき場所，入れるべきでない場所が決まっています．
引数に値を代入するときは空白を入れるべきではないです．